randovania/games/common/prime_family/gui/export_validator.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from pathlib import Path
+
+import nod
+
+from randovania.gui.dialog.game_export_dialog import is_file_validator
+
+
+def discover_game(iso_path: Path) -> tuple[str, str] | None:
+    if not iso_path.is_file():
+        return None
+
+    try:
+        disc, is_wii = nod.open_disc_from_image(iso_path)
+        data_partition: nod.Partition = disc.get_data_partition()
+        header: nod.DolHeader = data_partition.get_header()
+    except RuntimeError as e:
+        logging.warning("Unable to parse %s: %s", str(iso_path), str(e))
+        return None
+
+    return header.game_id.decode("UTF-8"), header.game_title.split(b"\x00")[0].decode("UTF-8")
+
+
+def is_prime1_iso_validator(file: Path | None) -> bool:
+    """Returns False when the given path exists and is either not an ISO or a Prime 1 ISO, True otherwise"""
```

Comment:
is this understanding correct? `Returns True if the given path either does not exist, or is a Prime 1 ISO; False otherwise`? if that's correct it would certainly prove easier (at least for mii) to read if it were written that way

Reply:
I really should change these to be `True` when it passes validation, other than not >.>

The validator succeds (aka False result) if the file is:
- exists, but not an iso
- exists, is ISO and Prime 1

else, fails (aka True)